### ISSUE TITLE
BUG: CC0052 (Make readonly) does not work with lambda expressions and initialized variables

### ISSUE THREAD

**Issue by kindermannhubert** [code-cracker/code-cracker#853]
Example:
```c#
    class C
    {
        private readonly Action set;
        private int i = 0;

        public C()
        {
            set = () => i = 1;
        }

        public void Modify()
        {
            set();
        }
    }
```
CC0052 is detected for "i = 0" but this kind of change would break compilation.

**Comment by carloscds** [comment]
@kindermannhubert Today we are not looking for this into CC0052. Maybe this make sense. What do you guess @giggio ?

**Comment by GGG-KILLER** [comment]
As I stated on #854 this kind of change will not break compilation.

**Comment by GGG-KILLER** [comment]
As I stated on #854 this kind of change [would not break compilation](https://repl.it/En1u)

**Comment by giggio** [comment]
Definitely a bug. It is ready and up for grabs.

**Comment by giggio** [comment]
Closed in #862
